hardware/kubernetes_test.go

```diff
@@ -0,0 +1,30 @@
+package hardware_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/tinkerbell/hegel/hardware"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+func TestKubernetesClient(t *testing.T) {
+	config, err := clientcmd.BuildConfigFromFlags("", "/Users/cpd/.kube/config")
```

Comment:
Can we use envtest instead of a hardcoded kubeconfig?

Reply:
This is a local test I put together for ease. Its going to be replaced entirely with unit tests and e2e tests. 

The e2e tests should address the [export comment](https://github.com/tinkerbell/hegel/pull/82#discussion_r851402168).

Reply:
Won't address on this PR. Reopen if needed.